script/update_dbdefs.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT_NAME=$(basename "$0")
+
+LIST_METABRAINZ_HOSTS_DEFAULT='../docker-server-configs/scripts/list_nodes.sh'
+
+HELP=$(cat <<EOH
+Usage: $SCRIPT_NAME [<hosts list>]
+
+Copy the latest DBDefs (as stored in docker-server-configs) to all MB
+containers on the specified hosts, or all MB service hosts if none are
+specified.
+
+It's expected that your SSH config contains host aliases identical to the
+ones you've passed, e.g.:
+
+  Host aretha
+    User bitmap
+    HostName aretha.metabrainz.org
+
+The DBDefs files will only be copied into the containers if they've changed.
+Services will be sent SIGHUP where applicable.
+
+Please make sure that your working copy of docker-server-configs is
+up-to-date beforehand!
+
+Environment:
+
+  LIST_METABRAINZ_HOSTS
+    Path to the script that lists MetaBrainz hosts by service.
+    Default: $LIST_METABRAINZ_HOSTS_DEFAULT
+
+  CONTAINER_SUFFIX
+    A "musicbrainz-" container suffix to limit changes to, like "json-dump"
+    or "webservice-test".
+    Default: (none)
+EOH
+)
+
+MB_SERVER_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/../" && pwd)
+
+cd "$MB_SERVER_ROOT"
+
+if [ $# -eq 0 ]; then
+  LIST_METABRAINZ_HOSTS=${LIST_METABRAINZ_HOSTS:-$LIST_METABRAINZ_HOSTS_DEFAULT}
+
+  if [ ! -x "$LIST_METABRAINZ_HOSTS" ]; then
+    echo >&2 "$SCRIPT_NAME: cannot list hosts"
+    echo >&2
+    echo >&2 "Please set \$LIST_METABRAINZ_HOSTS or specify the hosts list"
+    exit 69
+  fi
+  HOSTS=$("$LIST_METABRAINZ_HOSTS" musicbrainz | sort -u)
+else
+  if echo "$1" | grep -Eqx -- '-*h(elp)?'; then
+    echo "$HELP"
+    exit
+  fi
+  HOSTS="$*"
+fi
+
+last_host=${HOSTS/* }
+
+for host in $HOSTS; do
+  echo "$host: Updating containers..."
+  ssh "$host" \
+    sudo -H -S -- /root/docker-server-configs/scripts/update_dbdefs.sh $CONTAINER_SUFFIX
```

Comment:
From `shellcheck script/update_dbdefs.sh` (<https://www.shellcheck.net/wiki/SC2086>):

```suggestion
    sudo -H -S -- /root/docker-server-configs/scripts/update_dbdefs.sh "$CONTAINER_SUFFIX"
```

Reply:
That's intentional because if `$CONTAINER_SUFFIX` is empty, `"$CONTAINER_SUFFIX"` will pass an argument.  But open to suggestions there.